DownloadAllSongsAndAlbums.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+mkdir -p $HOME/Music/SongLibrary
+mkdir -p $HOME/Pictures/Covers
+cd $HOME/Music/SongLibrary
```

Comment:
I think good convention might be to encapsulate anything happening in `$HOME/Music/SongLibrary` in it's own subshell (Surround it with parenthesis), in the event that later we need to do something in the original directory.